cmd/kube-proxy/app/server.go

```diff
@@ -105,9 +109,26 @@ func (s *ProxyServer) Run(_ []string) error {
 	// are registered yet.
 
 	// define api config source
-	if s.ClientConfig.Host != "" {
-		glog.Infof("Using API calls to get config %v", s.ClientConfig.Host)
-		client, err := client.New(&s.ClientConfig)
+	if s.Master != "" || s.Kubeconfig != "" {
```

Comment:
I'm not really close to this part of the code, but if you don't have a `master` or a `kubeconfig`, what are you proxying to?